website/blog/2023-09-05-wrapper-as-a-standalone-component.md

```diff
@@ -0,0 +1,157 @@
+---
+title: How to use the Wrapper as a standalone component
+description: Explains the process of using the Wrapper component in WordPress templates.
+slug: wrapper-as-a-standalone-component
+authors: kancijan
+
+date: 2023-09-05
+tags: [eightshift, boilerplate, wrapper, components]
+hide_table_of_contents: false
+---
+
+As one of the most powerful features in the Eightshift DevKit, the Wrapper is a component used for every block in the Gutenberg editor, but what about WordPress templates?
+<!--truncate-->
+
+:::note
+[The Wrapper](https://eightshift.com/docs/basics/blocks-wrapper/) is designed to be the ultimate top-level component that controls how your block behaves in the website layout. The Wrapper is a sort of a 'section' in traditional builders. By default, you can control a whole lot of stuff, but there is an ability to add your custom attributes and fine-tune the Wrapper to your project's needs.
```

Comment:
```suggestion
[The Wrapper](https://eightshift.com/docs/basics/blocks-wrapper/) is designed to be the ultimate top-level component that controls how your block behaves in the website layout. The Wrapper is a sort of a 'section' in traditional builders. By default, you can control a whole lot of stuff, but there is an option to add your custom attributes and fine-tune the Wrapper to your project's needs.
```
I feel the word ability refers more to humans rather than objects :)

Reply:
Agree, but it's a direct quote from the wrapper docs. I will change it here though.